test/lib/data_gen.py

```diff
@@ -0,0 +1,44 @@
+from lib.helper import get_test_folder
+import json
+import jinja2
+import yaml
+import os
+
+_config_path = os.path.join(get_test_folder(), 'config.yaml')
+with open(_config_path, 'r') as yaml_file:
+    config = yaml.load(yaml_file)
+
+
+def generate_connector_content(input_disc=None):
+    default_disc = \
+        {
+            "name": "",
+            "connector_class": "com.splunk.kafka.connect.SplunkSinkConnector",
+            "tasks_max": "1",
+            "topics": config["kafka_topic"],
+            "splunk_indexes": config["splunk_index"],
+            "splunk_hec_uri": config["splunk_hec_url"],
+            "splunk_hec_token": config["splunk_token"],
+            "splunk_hec_raw": "false",
+            "splunk_hec_ack_enabled": "false",
+            "splunk_hec_ssl_validate_certs": "false",

```

Comment:
shouldn't we use `.` instead of `_`?

Reply:
These are the keys for data_gen so we can update the payload easily through the template. In the connector.template, they are all `.`

Reply:
approved. personally i think it is better to be identical. to reduce any possible confusion